coding-concepts.js

```diff
@@ -8,44 +8,44 @@
 // --------------------1) What will this log?
 
 var cohort = "Delta 2021"
-// console.log(cohort.length)
+ console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 9
+// b) Verify and explain: My answer was incorrect because I began counting at zero thinking this had to do with the position, but in fact had to do with the value instead.
 
 
 // --------------------2) What will this log?
 
 var greeting = "Hello World!"
-// console.log(greeting[3])
+console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: l
+// b) Verify and explain: My answer was correct because the square brackets are asking for the index of the letter in position 3 which is l.
 
 
 // --------------------3) What will this log?
 
 var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: J
+// b) Verify and explain: My answer was incorrect because I was focused on the character that would return an index of 1 when it had to do with the word.
 
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["SATURDAY","SUNDAY"]
+// b) Verify and explain: The code came back with an errorstating that "weekendDays.toUpperCase())" is not a function, but I'm certain it would return with my previous answer.
```

Comment:
So this a good moment where taking a good look at your code is important. This code was designed to fail in this way because .toUpperCase() doesn't work on arrays. yes if we wrote
```javascript
console.log(weekendDays[0].toUpperCase())
console.log(weekendDays[1].toUpperCase())
```
you'd be correct, but thats why we threw you this curve ball.